src/containers/organizations/components/CreateOrganizationComponent.js

```diff
@@ -95,12 +107,61 @@ class CreateOrganization extends React.Component {
 
   onClickCreate = () => {
 
+    const emailDomains = this.state.domains.map((obj) => {
+      return obj.value;
+    });
+
     const org :Organization = (new OrganizationBuilder())
       .setTitle(this.state.title)
       .setDescription(this.state.description)
+      .setAutoApprovedEmails(emailDomains)
       .build();
 
-    OrganizationsApi.createOrganization(org);
+    OrganizationsApi.createOrganization(org)
```

Comment:
The request logic should be moved into an `OrganizationEpic`. The epic should listen for actions of type  `ORGANIZATION_CREATE_REQUEST` (or whatever it's named), and that action should contain the organization to be created. See `CatalogEpic.searchCatalogEpic` for an example. It transforms the response into a `ORGANIZATION_CREATE_RESOLVE` or `ORGANIZATION_CREATE_REJECT` action, which is then picked up by the reducer. See `CatalogReducer` for an example.